### ISSUE TITLE
As a restricted platform developer I'd like a template for a blank restricted platform setup

### ISSUE THREAD

**Issue by AMZN-alexpete** [o3de/o3de#15519]
**Is your feature request related to a problem? Please describe.**
Developers of restricted platforms need an example of a fake restricted platform to use as an example for how to setup a real restricted platform for O3DE.

**Describe the solution you'd like**
1. A separate public repository is created that contains a fake restricted platform
2. Documentation exists explaining how to setup O3DE and the fake restricted platform repository and compile O3DE with this fake restricted platform

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by byrcolin** [comment]
This is going to change with the new schema 2.0.0 https://github.com/o3de/o3de/issues/17505 
TLDR: This will no longer be "restricted" but "platform" objects, so this will be blocked by that.
